src/bitwarden_workflow_linter/rules/run_actionlint.py

```diff
@@ -34,7 +35,7 @@ def install_actionlint(platform_system: str) -> Tuple[bool, str]:
     return False, error
 
 
-def install_actionlint_source(error) -> Tuple[bool, str]:
+def install_actionlint_source(error, settings = None) -> Tuple[bool, str]:
     """Install Actionlint Binary from provided script"""
     url = "https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash"
     version = "1.6.17"
```

Comment:
Was this intended to say `version` to be able to pass in a version override?

```suggestion
def install_actionlint_source(error, version = "1.6.17") -> Tuple[bool, str]:
    """Install Actionlint Binary from provided script"""
    url = "https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash"
```